test/test_client.py

```diff
@@ -936,6 +936,10 @@ def test_missing(self):
   def test_missing_non_strict(self):
     ok_(self.client.acl_status('foo', strict=False) is None)
 
+  def test_set_acl(self):
+    self.client.write('foo', 'hello, world!')
+    content = self.client.set_acl('foo', 'user::rwx,group::r--,other::---')
+    ok_(content == 'Success')
```

Comment:
Since we have both `acl_status` and `set_acl` methods, could you write a test setting the ACL spec then retrieving it and checking that it matches?